### PR TITLE
[zh]: Resync ephemeral-volumes.md

### PR DESCRIPTION
**Pull request by zaunist**

### CODE REVIEW

File: content/zh/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -202,14 +203,18 @@ driver. These attributes are specific to each driver and not
 standardized. See the documentation of each CSI driver for further
 instructions.
 
+### CSI driver restrictions
+
 As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
 [`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).
 -->
 `volumeAttributes` 决定驱动程序准备什么样的卷。这些属性特定于每个驱动程序，且没有实现标准化。
 有关进一步的说明，请参阅每个 CSI 驱动程序的文档。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/ephemeral-volumes.md:212]: 这两行放到 205 之后，保持每段翻译不要太大。

File: content/zh/docs/concepts/storage/ephemeral-volumes.md [hunk 2]
```diff
@@ -202,14 +203,18 @@ driver. These attributes are specific to each driver and not
...
 
-作为一个集群管理员，你可以使用 
-[PodSecurityPolicy](/zh/docs/concepts/policy/pod-security-policy/) 
+### CSI driver restrictions
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/ephemeral-volumes.md:214]:
```suggestion
### CSI 驱动限制   {#csi-driver-restrictions}
```

File: content/zh/docs/concepts/storage/ephemeral-volumes.md [hunk 3]
```diff
@@ -202,14 +203,18 @@ driver. These attributes are specific to each driver and not
...
+
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/ephemeral-volumes.md:215]: 这里漏掉了

```
{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
```

File: content/zh/docs/concepts/storage/ephemeral-volumes.md [hunk 4]
```diff
@@ -202,14 +203,18 @@ driver. These attributes are specific to each driver and not
...
+作为一个集群管理员，你可以使用
+[PodSecurityPolicy](/zh/docs/concepts/policy/pod-security-policy/)
 来控制在 Pod 中可以使用哪些 CSI 驱动程序，
 具体则是通过 [`allowedCSIDrivers` 字段](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy)
 指定。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/ephemeral-volumes.md:220]: 这段后面漏了 `{{note}}`段落。

**Comment by tengqm** [comment]
You can run the following command on the `main` branch to find out the changes to be synced:

```
./scripts/lsync.sh content/zh/docs/concepts/storage/ephemeral-volumes.md
```

**Comment by zaunist** [comment]
> 

Got it.

File: content/zh/docs/concepts/storage/ephemeral-volumes.md [hunk 5]
```diff
@@ -202,18 +203,41 @@ driver. These attributes are specific to each driver and not
...
+{{< note >}}
+CSI ephemeral volumes are only supported by a subset of CSI drivers.
+The Kubernetes CSI [Drivers list](https://kubernetes-csi.github.io/docs/drivers.html)
+shows which drivers support ephemeral volumes.
+{{< /note >}}
+-->
+
+{{< note >}}
+CSI 临时卷仅有 CSI 驱动程序的一个子集支持。
+Kubernetes CSI [驱动列表](https://kubernetes-csi.github.io/docs/drivers.html) 显示了哪些驱动程序支持临时卷。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/ephemeral-volumes.md:238]:
```suggestion
Kubernetes CSI [驱动列表](https://kubernetes-csi.github.io/docs/drivers.html)显示了哪些驱动程序支持临时卷。
```

Comment (zaunist) [reply content/zh/docs/concepts/storage/ephemeral-volumes.md:238]: Thanks for your carefulness.

**Comment by howieyuen** [comment]
@zaunist oops... PR has a conflict, please rebase from the main branch

**Comment by zaunist** [comment]
> /label tide/merge-method-squash

Thanks, but this PR  also requires `lgtm` label to merge.

**Review by tanjunchen — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**